client/post-editor/post-editor.jsx

```diff
@@ -1136,7 +1111,6 @@ export default connect(
 	},
 	( dispatch ) => {
 		return bindActionCreators( {
-			editPost,
```

Comment:
`editPost` is duplicated on master as well so when I rebased my PR I pulled in the duplicate which triggered an error and build failure. I removed the duplicate (`editPost` is also in line 1143.)